plugins/main/public/components/overview/vulnerabilities/dashboards/overview/dashboard_panels.ts

```diff
@@ -431,7 +431,7 @@ const getVisStateAccumulationMostDetectedVulnerabilities = (
           enabled: true,
           type: 'date_histogram',
           params: {
-            field: 'event.created',
+            field: '@timestamp',
```

Comment:
**thought**: the index pattern of alerts defines the `timestamp` as time filed. Should the vulnerability detection states use the same field to keep the consistency?